server/sandbox_status_test.go

```diff
@@ -0,0 +1,120 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/kubernetes-incubator/cri-o/lib"
+	"github.com/kubernetes-incubator/cri-o/lib/sandbox"
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/version"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
+func newTestContainerServerOrFailNow(t *testing.T) (cs *lib.ContainerServer, dirsToCleanUp []string) {
+	tmpdir := os.Getenv("TMPDIR")
+
+	config := lib.DefaultConfig()
+	runRoot, err := ioutil.TempDir(tmpdir, "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	config.RootConfig.RunRoot = runRoot
+	root, err := ioutil.TempDir(tmpdir, "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	config.RootConfig.Root = root
+	config.RootConfig.Storage = "vfs"
+	cs, err = lib.New(config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return cs, []string{runRoot, root}
+}
+
+func newTestSandboxOrFailNow(t *testing.T) (string, *sandbox.Sandbox) {
+	id := fmt.Sprintf("id-for-sandbox-%d", rand.Int())
+
+	sb, err := sandbox.New(id, "", "", "", "", nil, nil, "", "", nil, "", "", false, false, "", "", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return id, sb
+}
+
+func newTestContainerOrFailNow(t *testing.T) (string, *oci.Container) {
+	id := fmt.Sprintf("id-for-container-%d", rand.Int())
+
+	c, err := oci.NewContainer(id, "", "", "", nil, nil, nil, nil, "", "", "", nil, "", false, false, false, false, false, "", time.Now(), "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	return id, c
```

Comment:
No need to return `id` here, because you can get it with [`c.ID()`][1].

[1]: https://github.com/kubernetes-incubator/cri-o/blob/74cd1ec97c13a9784ce5e67a9e50e8977b5d2f38/oci/container.go#L155-L158